src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,417 @@
-possible, the transition from backup to primary should be managed formally.
-Otherwise, the backup should simply listen in but otherwise not engage.
-
-Other Developer's Role
-----------------------
-
-In the ideal, the primary is able to handle all customer response activity and
-no other developers are interrupted. Thus, other developers should feel free to
-ignore customer inquiries and/or redirect customers who may contact them directly.
-One option to reduce visit-user traffic is to switch to receiving a daily digest.
-Other options might be email client side filters.
+  * Identifying a *low-hanging fruit* type issue and subsequently engaging
+    in the *housekeeping* work to resolve it.
+  * Determining if the user's issue is known (e.g. an issue ticket already exists).
+  * Updating a known issue with new information from this user, perhaps
+    adjusting labels on the issue or putting the issue back into the
+    UN-reviewed state for further discussion at a VisIt_ project meeting.
+  * Identifying and filing a new *product development* type issue ticket.
+
+To emphasize the last bullet above, *resolution* of a **Customer Response**
+issue does not always mean a customer's issue can be addressed to *satisfaction*
+within the constraints of the processes defined here. Sometimes, the most
+that can be achieved is filing a highly informative issue ticket to be
+prioritized, scheduled and ultimately resolved as part of normal VisIt_ product
+development activities. Nonetheless, doing so does serve to *resolve* the
+original **Customer Response** issue that initiated the work.
+
+Handoffs
+--------
+
+Our **Customer Response** processes involve two kinds of *handoffs*. One is the
+redirection of a customer who makes contact with a developer not serving as the
+**Primary**. The other is the handoff of unresolved **Customer Response** issues
+from one week's **Primary** to the next.
+
+To handle customer redirection handoffs, it is a best practice to use a three-way
+handoff giving the customer some assurance that their initial contact with someone
+is successfully handed off to the **Primary**. For example, for a call-in, it
+is a best practice to try a three-way call transfer. The propspect of redirecting
+friends and colleagues with whome some developers may have long standing relatinships
+may be initially uncomfortable. But it is important to recognize that this an
+essential part of achieving the goals of this process.
+
+If an active **Customer Response** issue cannot be resolved within the week of
+a **Primary**'s assignment, it gets handed off to the next week's **Primary**.
+Such handoffs shall be managed formally with a comment (or email) to the
+customer(s) and the next week's **Primary** and **Backup** in the associated
+GitHub issue. The associated issue(s) in the **Customer Response** issues
+repository shall be re-assigned by the next week's **Primary** upon beginning
```

Comment:
Good point. Will add a line about negotiating this with nex week's primary.

Reply:
> It sounded like these would be added to a new issue tracker. It seems like adding a new label to any existing tickets would be better, since this is simpler, keeps a single source of issues and is just as functional as a new issue tracker.

Yes, new label, not a new tracker. Will clarify.

Reply:
> Suppose the primary of week 1 is almost done resolving an issue that came in, but the week ends. The primary wants to continue working on it Monday morning because it will only take a few more hours to complete, the same amount of time that it would take the new primary for week 2 to get up to speed on the issue. How do we deal with these types of situations?

I have added verbiage of the primary's option to do this. I agree it make sense that sometimes a primary will do this.